src/features/device-state/routes/state-get-v3.ts

```diff
@@ -298,6 +310,13 @@ const getStateV3 = async (req: Request, uuid: string): Promise<StateV3> => {
 		},
 	};
 
+	for (const depDev of device.manages__device as AnyObject[]) {
+		state[depDev.uuid] = {
+			name: depDev.device_name,
+			apps: getUserAppState(depDev, getConfig(depDev)),
+		};
+	}
+
```

Comment:
Can you revert this - I'm pretty sure it's a failed rebase artifact
```suggestion
```

Reply:
done.